tests/acceptance/help-test.js

```diff
@@ -9,7 +9,7 @@ const processHelpString = require('../helpers/process-help-string');
 const convertToJson = require('../helpers/convert-help-output-to-json');
 const commandOptions = require('../factories/command-options');
 const HelpCommand = require('../../lib/commands/help');
-const requireAsHash = require('../../lib/utilities/require-as-hash');
+const requireAsHash = require('../../lib/utilities/require-as-hash').default;
 const Command = require('../../lib/models/command');
 
```

Comment:
Since your tests are running with `node -r esm` this test file could be written in ESM too ☝️